strava/client.py

```diff
@@ -261,18 +261,24 @@ def get_segment(self, segment_id):
         path = f'segments/{segment_id}'
         return self._dispatcher('get', path)
 
-    def get_segment_efforts(self, segment_id, per_page=50, limit=None):
+    def get_segment_efforts(self, segment_id, per_page=50, limit=None, start_date=None, end_date=None):
         """
         Return all segment's efforts from activities of the authenticated user.
 
         See docs: http://developers.strava.com/docs/reference/#api-SegmentEfforts-getEffortsBySegmentId
 
         :param segment_id [int]: Segment id.
+        :param start_date [Date]: Segment start date.
+        :param end_date [Date]: Segment end date.
```

Comment:
`start_date` and `end_date` should be a datetime instead of date. Then I'd name it as `start_dt`, `end_dt` and update the docstring